runbotics/runbotics-orchestrator-ui/src/src-app/components/tables/NotificationTable/NotificationTableComponent.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { ChangeEvent, VFC, useMemo, useState } from 'react';
+
+import { Grid, Typography } from '@mui/material';
+import { DataGrid, GridColDef } from '@mui/x-data-grid';
+
+import styled from 'styled-components';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+import { StyledHeaderGrid } from '#src-app/views/users/UsersBrowseView/UsersBrowseView.styles';
+import { DefaultPageValue } from '#src-app/views/users/UsersBrowseView/UsersBrowseView.utils';
+import { StyledActionsContainer, StyledTextField } from '#src-app/views/users/UsersListView/UsersListView.styles';
+
+import { BotNotificationRow, ProcessNotificationRow } from './NotificationTableComponent.types';
+
+const StyledWrapper = styled.div`
```

Comment:
Mikołaj told me when I'll create new styles it should be in .styles.ts file, but I'm not sure if just one Wrapper needs it - your decision.